<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QPushButton" name="back">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>უკან დაბრუნება</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="calculate">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>გამოთვლა</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="main">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="4" column="2">
         <widget class="QRadioButton" name="kvadrati">
          <property name="text">
           <string>კვადრატი</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">shape_group</string>
          </attribute>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QPushButton" name="select">
          <property name="text">
           <string>არჩევა</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="2">
         <widget class="QRadioButton" name="trapecia">
          <property name="text">
           <string>ტრაპეცია</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">shape_group</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QRadioButton" name="wre">
          <property name="text">
           <string>წრე</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">shape_group</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QRadioButton" name="samkutxedi">
          <property name="text">
           <string>სამკუთხედი</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">shape_group</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="samkutxedi_page">
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>126</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>გვერდი1</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="s1">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>120</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>180</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>გვერდი2</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="s2">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>170</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>230</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>გვერდი3</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="s3">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>220</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>320</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>ფართობი</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>360</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>პერიმეტრი</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcd_s_area">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>310</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcd_s_perimeter">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>350</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>0</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>სამკუთხედის კალკულაცია</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="wre_page">
       <widget class="QDoubleSpinBox" name="r1">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>110</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>126</x>
          <y>116</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>რადიუსი</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>200</y>
          <width>121</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>ფართობი</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>260</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>სიგრძე</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcd_r_perimeter">
        <property name="geometry">
         <rect>
          <x>213</x>
          <y>252</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>0</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>წრის კალკულაცია</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcd_r_area">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>190</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="trapecia_page">
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>10</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ტრაპეციის კალკულაცია</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="st1">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>70</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="st2">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>120</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="st3">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>170</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="st4">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>220</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>80</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>გვერდი1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>220</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>გვერდი4</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>130</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>გვერდი2</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>170</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>გვერდი3</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>350</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>პერიმეტრი</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcd_tr_area">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>300</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcd_tr_perimeter">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>340</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>310</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>ფართობი</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="kvadrati_page">
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>10</y>
          <width>181</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>კვადრატის კალკულაცია</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="s_kv">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>60</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>190</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>პერიმეტრი</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcd_kv_area">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>140</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcd_kv_perimeter">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>180</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>150</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>ფართობი</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="errorLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="shape_group"/>
 </buttongroups>
</ui>
